stack/rest/src/main/java/org/apache/usergrid/rest/organizations/OrganizationResource.java

```diff
@@ -143,21 +143,21 @@ public ApplicationResource getApplicationByName( @PathParam("applicationName") S
     }
 
 
-    @Path("applications/{applicationName}")
+    @Path("applications/{applicationName: [^<>/]+}")
```

Comment:
I tested this out locally, not sure if it's doing what you'd expect.  I went to this URL (my local image shows 8580 as the HTTP port)

http://localhost:8580/applications/%3Cscript%3Ealert('bob')%3C/script%3E

The response still included the script tag, so this came back

```

{"error":"organization_application_not_found","timestamp":1436323964157,"duration":3,"exception":"org.apache.usergrid.rest.exceptions.OrganizationApplicationNotFoundException","error_description":"Could not find application for applications/<script>alert('bob')< from URI: applications/<script>alert('bob')</script>"}
```